docs/en/operations/system-tables/information_schema.md

```diff
@@ -209,6 +209,7 @@ Columns:
   number of rows. NULL if it could not be determined.
 - `data_length` ([Nullable](../../sql-reference/data-types/nullable.md)([UInt64](../../sql-reference/data-types/int-uint.md))) — The size of
   the data on-disk. NULL if it could not be determined.
+- `index_length` ([Nullable](../../sql-reference/data-types/nullable.md)([UInt64](../../sql-reference/data-types/int-uint.md))) — The total size of all the compressed indexes, marks, compressed secondary indexes, and their marks. This column is computed from the contents of the [system.parts](../../operations/system-tables/parts.md) table.
```

Comment:
Maybe more clear stating the total size of primary key, secondary indexes and their marks?